s3prl/downstream/asr/config.yaml

```diff
@@ -41,9 +41,11 @@ downstream_expert:
     test-other: ['test-other']
     num_workers: 12
     train_batch_size: 32
+    batch_size: 32
     eval_batch_size: 1
-    libri_root: '/path/to/LibriSpeech'
-    bucket_file: 'data/librispeech/len_for_bucket'
+    libri_root: '/app/data/LibriSpeech'
```

Comment:
We use each task's `config.yaml` file to define the various filepaths for the data, where each path is defined relative to the Docker container's filesystem